lib/gds_api/config.rb

```diff
@@ -0,0 +1,17 @@
+module GdsApi
+  def self.configure
+    yield(config)
+  end
+
+  def self.config
```

Comment:
Because then you would have to set it for all clients. Is that something you would prefer?